packages/gatsby-theme-carbon/src/components/Code/Code.js

```diff
@@ -33,15 +43,11 @@ const Code = ({ children, className: classNameProp = '', path, src }) => {
   const getLines = (lines) => {
     const withoutTrailingEmpty = removeTrailingEmptyLine(lines);
 
-    if (withoutTrailingEmpty.length > 9) {
-      setHasMoreThanNineLines(true);
-    }
-
-    if (shouldShowMore) {
+    if (isOverflowExpanded) {
```

Comment:
Need to normalize the capitalization between variagbles `OverFlow` vs `Overflow`

Reply:
🤦 Nice catch